src/data/data.test.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from "prop-types";
+
+import {getFaction} from "../util/jsonUtils";
+import {unitTemplate} from "./dataTemplates";
+
+const originalConsoleError = console.error;
+
+beforeAll(() => {
+  console.error = message => {
```

Comment:
Overwrite console.error for this testfile, the hacky part. Is there a better/ more correct way to do this?

Reply:
Could you not Sinon/sandbox stub this?

Reply:
Using jest stubbing! Good ideas